mars/services/cluster/api/oscar.py

```diff
@@ -320,6 +320,17 @@ async def fetch_node_log(
         ref = await self._get_log_ref(address)
         return await ref.fetch_logs(size, offset)
 
+    async def _get_yaml_dumper_ref(self, address: str):
```

Comment:
How about moving the interface to TaskService since this interface is not relevant to cluster management?